win32/ctags_vs2013.vcxproj

```diff
@@ -367,4 +368,4 @@
   <Import Project="$(VCTargetsPath)\Microsoft.Cpp.targets" />
   <ImportGroup Label="ExtensionTargets">
   </ImportGroup>
-</Project>
\ No newline at end of file
+</Project>

```

Comment:
Don't include this change in the pull request.
You can add parts to your commit with 'git commit -p'.

Reply:
Removed